app/models/container_image.rb

```diff
@@ -97,5 +97,10 @@ def disconnect_inv
     save
   end
 
+  def self.disconnect_inv(ids)
+    _log.info "Disconnecting ContainerImages ids [#{ids}]"
```

Comment:
Can you more closely match the log message from the normal [disconnect_inv](https://github.com/Ladas/manageiq/blob/2c10dbfa820f7b003d3a6eeb19da6e6b5583dcce/app/models/container_image.rb#L93) ?
Maybe `"Disconnecting Images [#{ids}] from EMS [#{ext_management_system.name}] id [#{ext_management_system.id}]"`